src/Model/GServer.php

```diff
@@ -117,6 +117,31 @@ public static function getID(string $url, bool $no_check = false)
 		return self::getID($url, true);
 	}
 
+	/**
+	 * Retrieves all the servers which base domain are matching the provided domain pattern
+	 *
+	 * The pattern is a simple fnmatch() pattern with ? for single wildcard and * for multiple wildcard
+	 *
+	 * @param string $pattern
+	 * @return array
+	 * @throws Exception
+	 */
+	public static function listByDomainPattern(string $pattern): array
+	{
+		$likePattern = 'http://' . strtr($pattern, ['_' => '\_', '%' => '\%', '?' => '_', '*' => '%']);
+
+		$sql = "SELECT s.*, COUNT(*) AS `contacts`
+			FROM `gserver` s
+			LEFT JOIN `contact` c ON s.`id` = c.`gsid`
```

Comment:
I'm not a huge fan of this `s` and `c` naming.

Reply:
We've had this discussion in the past, I feel like repeating the name of the table is too verbose, and then aliases are needed in self-joins anyway, so I've taken the habit of always using them.

Reply:
For me it decreases the readability. With the same argument we could name our variables in the code from `$a` to `$z`. When using the same table multiple times I also prefer more speaking aliases.

Reply:
You know very well that I don't name the variables I create this short so you're perfectly aware that I do not use this argument indiscriminately. However, if you're ready to argue in bad faith like this, I guess this is an important subject for you and I have made the necessary changes.

Reply:
At first: Thanks for the change! 

My argument just was that since we don't use short variable names we shouldn't do so in queries. I don't see (and mean) this as "argue in bad faith".

Reply:
For me it's an entirely different context. Table aliases can be short because the actual table name is also present in the same query. If it was something like `SELECT s.id FROM s WHERE s.url = ?` then I would agree, it would be the same issue with short variable names. But the full name of the tables are also present so the alias is there just to prevent ambiguity, both for the SQL compiler and humans alike. Since they have a limited role, their size can be limited too.